routes/api.js

```diff
@@ -69,4 +71,9 @@ router.use('/swagger', DocsController.swagger);
 // Info
 router.get('/info', InfoController.sendInfo);
 
+// Admin
+router.post('/inviteCode', AdminMiddleware, InviteCodeController.addInviteCode);
+router.delete('/inviteCode/:inviteCode', AdminMiddleware, InviteCodeController.deleteInviteCode);
```

Comment:
you can also parse two middlewares. the `ProtectMiddleware` and your admin middleware. So you dont need to fetch the user. See comment in `AdminMiddleware.js` for more infos. *(dont forget to also change that in the other admin protected routes.)*
```suggestion
router.delete('/inviteCode/:inviteCode', ProtectMiddleware, AdminMiddleware, InviteCodeController.deleteInviteCode);
```

Reply:
Thats right, but I need to fetch the user anyway in order to check if his ``adminRights`` are true or false